tests/test_all_rules.py

```diff
@@ -814,3 +815,27 @@ def test_note_guide(self):
                 if osquery_note_pattern not in rule.contents.data.note:
                     self.fail(f'{self.rule_str(rule)} Investigation guides using the Osquery Markdown must contain '
                               f'the following note:\n{osquery_note_pattern}')
+
+
+class TestEndpointQuery(BaseRuleTest):
+    """Test endpoint-specific rules."""
+
+    def test_os_and_platform_in_query(self):
+        """Test that all endpoint rules have an os defined and linux includes platform."""
+        for rule in self.production_rules:
```

Comment:
If we only iterate over production rules, we'll miss out on rules with `maturity = "development"`.  Since we're already tuning them e.g. `defense_evasion_msbuild_beacon_sequence.toml`, we probably just expand to all rules.

Reply:
the whole point of rules in development is to not force compliance while they are in dev